components/cosmetic_filters/resources/data/element_picker.html

```diff
@@ -1,165 +1,317 @@
+      --text-color: #e3e3e8;
     }
+  }
 
-    #main-section {
-      background-color: #FFFFFF;
-      border-radius: .6em;
-      border: 1px solid #aaa;
-      color-scheme: light only;
-      bottom: 4px;
-      left: unset;
-      right: 4px;
-      width: calc(30% - 4px);
-      box-sizing: border-box;
-      min-width: 24em;
-      padding-top: 16px;
-      padding-right: 16px;
-      padding-left: 16px;
-      position: fixed;
-      opacity: 1;
-      height: auto;
-    }
+  .light {
+    --primary: #3E37D4;
+    --title-bar: #f0f0f4;
+    --block-btn-bg-disabled: #46464a33;
+    --block-btn-color-disabled: #21272a80;
+    --button-text: white;
+  }
 
-    @media (orientation: portrait) {
-      #main-section:not(.desktop) {
-        bottom: 0;
-        right: 0;
-        left: 0;
-        border-radius: unset;
-        border: none;
-        width: 100%;
-      }
-    }
-  
-    #main-section input {
-      width: calc(100% - 4px);
-    }
-  
-    #buttons {
-      display: flex;
-      justify-content: space-between;
-      flex-direction: column;
-    }
-  
-    .buttons-line {
-      display: flex;
-      justify-content: space-between;
-      flex-direction: row;
-      padding-bottom: 16px;
-    }
-  
-    .button {
-      min-height: 36px;
-      justify-content: center;
-      align-items: center;
-      border-radius: 8px;
-      font-size: 12px;
-      font-weight: 600;
-      flex-grow: 1;
-      border-width: 0px;
-    }
+  .dark {
+    --primary: #C1C4FF;
+    --title-bar: #303034;
+    --block-btn-bg-disabled: #ebeef033;
+    --block-btn-color-disabled: #ebeef080;
+    --button-text: #251491;
+  }
 
-    .block-button.light {
-      background-color: #3E37D4;
-      color: #FFFFFF;
-    }
-    .block-button.dark {
-      background-color: #C1C4FF;
-      color: #251491;
-    }
+  #rules-box {
+    height: 8em;
+    margin-bottom: 5px;
+    background-color: #fff;
+    border-radius: .1em;
+    display: none;
+  }
 
-    .block-button.disabled.light {
-      background: rgba(70,70,74,0.20);
-      color: rgba(33,39,42,0.50);
-    }
-    .block-button.disabled.dark {
-      background: rgba(235,238,240,0.20);
-      color: rgba(235,238,240,0.50);
-    }
+  .desktop.minimized>#rules-box {
+    display: none !important;
+  }
 
-    .secondary-button {
-      background-color: #FFFFFF;
-      color: #4036d3;
-      cursor: pointer;
-      padding: 4px 6px;
-      flex-grow: 1;
-      border-width: 0px;      
-    }
-    
-    .secondary-button.dark {
-      color: #c1c4ff;
-    }
+  #rules-box>textarea {
+    box-sizing: border-box;
+    height: 100%;
+    width: 100%;
+    resize: none;
+    border: none;
+    padding: 5px;
+    overflow: auto;
+    overflow-wrap: break-word;
+    background-color: white;
+    color: var(--text-color);
+  }
 
-    .secondary-button.light {
-      color: #3E37D4
-    }
-    .secondary-button.disable.dark {
-      color: #c1c4ff;
-    }
+  #main-section.dark>#rules-box>textarea {
```

Comment:
is this meant to be behind the `.dark` selector? Either way, let's remove `#main-section`

Additionally, it'd be nice if we didn't need the child selector and could just set the background on `#rules-box` (but that might not be possible if the `textarea` has a default background).